### PR TITLE
ZBUG-2363 Modern UI, message body displaying partially

### PR DESCRIPTION
**Pull request by subrahmanyaShetty090**
-Added truncated property in mime parts

### CODE REVIEW

**Review by damini-vashishtha — APPROVED** [review]

File: src/schema/schema.graphql [hunk 1]
```diff
@@ -1975,6 +1975,7 @@ type MimePart {
 	url: String
 	messageId: ID
 	base64: String
+	truncated:Boolean
```
Comment (silentsakky) [inline src/schema/schema.graphql:1978]: space after colon

Comment (subrahmanyaShetty090) [reply src/schema/schema.graphql]: implemented this change.

**Review by silentsakky — APPROVED** [review]

File: src/normalize/entities.ts [hunk 2]
```diff
@@ -15,7 +15,8 @@ const MimePart = new Entity({
 	s: 'size',
 	part: 'part',
 	mid: 'messageId',
-	content: 'content'
+	content: 'content',
+	truncated: 'truncated'
```
Comment (silentsakky) [inline src/normalize/entities.ts:19]: we don't need to add these if key/value are same

Comment (silentsakky) [reply src/normalize/entities.ts:19]: also remove part key from line no 16

Comment (subrahmanyaShetty090) [reply src/normalize/entities.ts]: removed these key value pair.

**Review by silentsakky — APPROVED** [review]

**Pull request merged**